taf/api/utils/roles_utils.py

```diff
@@ -75,6 +76,20 @@ def create_delegations(
         )
 
 
+@log_on_start(DEBUG, "Finding roles of key", logger=taf_logger)
+def get_roles_and_paths_of_key(
+    public_key: Dict,
+    repository: AuthenticationRepository,
+):
+    repository.find_associated_roles_of_key(public_key)
+    roles = ["targets", "law", "docs", "assets", "rdf"]
```

Comment:
Should taf be aware of platform delegated roles, such as `law`, `assets`, `docs`, and `rdf`? Can we make `get_roles_and_paths_of_key` work without specifying the delegated role names?

Reply:
Definitely not, I must have been very tired when I wrote this. good catch

Reply:
Ah, I remember now. I set that it order to be able to test the remainder of the function without a YK and forgot to remove.  `roles` should be assigned to `repository.find_associated_roles_of_key(public_key)`